ArduCopter/flight_mode.cpp

```diff
@@ -427,3 +427,15 @@ void Copter::print_flight_mode(AP_HAL::BetterStream *port, uint8_t mode)
     }
 }
 
+void Copter::zero_throttle_and_relax_ac()
+{
+#if FRAME_CONFIG == HELI_FRAME  // Helicopters always stabilize roll/pitch/yaw
+        // call attitude controller
+        attitude_control.input_euler_angle_roll_pitch_euler_rate_yaw(0, 0, 0, get_smoothing_gain());
+        attitude_control.set_throttle_out(0,false,g.throttle_filt);
+#else
+        motors.set_desired_spool_state(AP_Motors::DESIRED_SPIN_WHEN_ARMED);
+        // multicopters do not stabilize roll/pitch/yaw when disarmed
+        attitude_control.set_throttle_out_unstabilized(0,true,g.throttle_filt);
```

Comment:
Nitpicking (but since we are moving code around):

- put space between the parameters
- change the multicopter comment to be in the same line as the else
- remove the *call attitude controller* comment, it's completely redundant